game-core/src/main/java/games/strategy/engine/framework/startup/mc/SetupPanelModel.java

```diff
@@ -28,15 +29,17 @@
  * This class provides a way to switch between different ISetupPanel displays.
  */
 @RequiredArgsConstructor
-public class SetupPanelModel {
+public class SetupPanelModel implements ServerSetupModel {
   @Getter
   protected final GameSelectorModel gameSelectorModel;
   protected SetupPanel panel = null;
 
   @Setter
   private Consumer<SetupPanel> panelChangeListener;
+  private final JFrame ui;
```

Comment:
Should we annotate this as `@Nonnull` so `@RequiredArgsConstructor` generates a `null` check?  (It's not clear to me right now if it's always the case that usage of this class implies `ui` to be non-`null`, but it seems that way.)

Reply:
Good idea 👍